### ISSUE TITLE
pi0 lack of params when loading from pretrained model

### ISSUE THREAD

**Issue by OpenJarvisAI** [huggingface/lerobot#1461]
### System Info

```Shell
master branch
```

### Information

- [x] One of the scripts in the examples/ folder of LeRobot
- [x] My own task or dataset (give details below)

### Reproduction

the pi0 model loaded from pretrained model, lanauge.embed_tokens.weight is actually missing.

From I can tell, the whole model actually can not be loaded directly.

If you are finetuning pi0, with transformers >= 4.52.0, you will have a loss at 0.002, but the policy won't actually work

### Expected behavior

the model should loaded properly

**Comment by pkooij** [comment]
Hi we update our pi0 implementation and released pi05 support in this PR: https://github.com/huggingface/lerobot/pull/1910 🚀 . If you still have this issue please reopen this thread, but I am closing it for now.
